adapters/sspbc/sspbc.go

```diff
@@ -189,45 +145,50 @@ func (a *SspbcAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRe
 
 	for _, seatBid := range response.SeatBid {
 		for _, bid := range seatBid.Bid {
-			var BidType openrtb_ext.BidType
-			var BidId = bid.ImpID
+			var bidType openrtb_ext.BidType
+			var bidId = bid.ImpID
 
 			/*
-			  here we should make a call to getBidType method, and based on detected type
-			  make call to createBannerAd, createVideoAd, createNativeAd methods
+			  Determine bid type
+			  At this moment we only check if bid contains Adm property
 
-			  for now we set it to "banner"
+			  Later updates will check for video & native data
 			*/
-			BidType = openrtb_ext.BidTypeBanner
+			if (bid.AdM != "") { 
+				bidType = openrtb_ext.BidTypeBanner
+			}
+
 
-			if BidExt, ok := a.adSlots[BidId]; ok {
+			if BidExt, ok := a.adSlots[bidId]; ok {
 				var BidIdStored = BidExt.PbSlot
 				bid.ImpID = BidIdStored
-			} else {
-				glog.Errorf("SSPBC: BidExt for this bid.impid not found - %s", BidId)
 			}
 
 			// read additional data from proxy
 			var BidDataExt SsbcResponseExt
 			if err := json.Unmarshal(bid.Ext, &BidDataExt); err != nil {
-				glog.Errorf("SSPBC: cannot unmarshal Bid Ext data")
 				errors = append(errors, err)
 			} else {
 				var adCreationError error
 
-				// Prepare ads (using different methods for banner, native, video)
-
-				// BANNER
-				bid.AdM, adCreationError = a.createBannerAd(bid, BidDataExt, internalRequest, seatBid.Seat)
+				/*
+					use correct ad creation method for a detected bid type
+					right now, we are only creating banner ads
+					if type is not detected / supported, throw error
+				*/
+				if (bidType == openrtb_ext.BidTypeBanner) { 
+					bid.AdM, adCreationError = a.createBannerAd(bid, BidDataExt, internalRequest, seatBid.Seat)
+				} else {
+					adCreationError = fmt.Errorf("Bid type is not supported: %s", bidType)
```

Comment:
Linter: error strings should not be capitalized (ST1005) -> fmt.Errorf("bid type is not supported: %s", bidType)